### ISSUE TITLE
Watchify takes longer and longer with sequential changes

### ISSUE THREAD

**Issue by jlem** [laravel/elixir#177]
Have updated to the latest version of elixir via `npm update`

In my gulp file, I have this:

```javascript
elixir(function(mix) {
    mix.less('app.less');
    mix.browserify('app.js');
    mix.browserify('post.js', 'public/js/post.js');
    mix.angular('resources/assets/angular/', 'public/js/angular', 'application.js');
});
```

Whenever I add or update a new JS file or template file in one of my JS asset folders, watch takes longer and longer to finish. You have to Ctrl+C out of the watch and restart it once it starts getting too long.

I traced the bug to this issue: https://github.com/substack/watchify/issues/187

It appears that you shouldn't be running transforms inside of the bundle function:

https://github.com/laravel/elixir/blob/master/ingredients/browserify.js#L78

**Comment by rrosiek** [comment]
This PR should fix that: #176.

**Comment by JeffreyWay** [comment]
Merged
